src/ansible/roles/ipasmartcard/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+- block:
+  ################# IPA Server ####################
+  - name: Run IPA advise script for server setup for smart cards
+    shell: |
+      kinit admin@IPA.TEST
+      ipa-advise config-server-for-smart-card-auth > /tmp/sc_server.sh
+      sh -x /tmp/sc_server.sh {{ sssd_ca_path }} {{ ipa_ca_path }}
+    args:
+      stdin: '{{ ipa_password }}'
+    register: ipa_advise_sc_server
+
+  - name: Workaround to disable IPA WebUI OCSP checking
+    lineinfile:
+      path: /etc/httpd/conf.d/ssl.conf
+      regexp: 'SSLOCSPEnable on'
+      line: 'SSLOCSPEnable on no_ocsp_for_cert_ok'
+      backrefs: yes
+
+  - name: Restart httpd service
+    ansible.builtin.systemd_service:
+      name: httpd
+      state: restarted
+
+  when: inventory_hostname == 'master.ipa.test'
+
+- block:
+  ################# IPA Client ####################
+  - name: Get IPA advise script for client setup for smart cards from IPA server
+    shell: |
+      ipa-advise config-client-for-smart-card-auth
+    delegate_to: "master.ipa.test"
```

Comment:
Another hardcoded  host.  If we want to setup client with first ipa it would be better to reference it though group ipa nad index instead.